Makefile

```diff
@@ -50,7 +50,8 @@ CH5xx_ble_firmware_library/StdPeriphDriver/CH58x_usb2hostBase.c \
 CH5xx_ble_firmware_library/StdPeriphDriver/CH58x_spi1.c \
 CH5xx_ble_firmware_library/RVMSIS/core_riscv.c \
 src/main.c \
-src/leddrv.c
+src/leddrv.c \
+src/fb.c
```

Comment:
you can end all entries with \ , it will just merge with the following blank line